base/docs/basedocs.jl

```diff
@@ -1181,6 +1199,12 @@ end
     Symbol(x...) -> Symbol
 
 Create a `Symbol` by concatenating the string representations of the arguments together.
+
+# Examples
+```jldoctest
+julia> Symbol("my", "name")
```

Comment:
Just a suggestion; To highlight the "string representations of the arguments" part better perhaps not use two `String`s?